templates/zigbee2mqtt.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>Zigbee2MQTT</Name>
+  <Repository>koenkk/zigbee2mqtt</Repository>
+  <Registry>https://hub.docker.com/r/koenkk/zigbee2mqtt/</Registry>
+  <Network>bridge</Network>
+  <Privileged>false</Privileged>
+  <Support>https://www.zigbee2mqtt.io/information/docker.html</Support>
+  <Project>https://www.zigbee2mqtt.io/</Project>
+  <Overview>Zigbee to MQTT bridge using zigbee-shepherd allows you to use your Zigbee devices without the vendors bridge or gateway. It bridges events and allows you to control your Zigbee devices via MQTT. In this way you can integrate your Zigbee devices with whatever smart home infrastructure you are using. Converted By Community Applications   Always verify this template (and values) against the dockerhub support page for the container</Overview>
```

Comment:
This will probably still be rejected by CA

Reply:
Hey @Roxedus mind being more specific on which part makes it rejected? I changed the overview to not have characters that are html encoded, but now i do not understand what is missing or needs to be removed.

Reply:
The fact that it says converted from CA, and everything related